scripts/g.extension/g.extension.py

```diff
@@ -1405,15 +1418,23 @@ def install_extension_std_platforms(name, source, url):
 
     # collect module names
     module_list = list()
+    module_name_not_found = _("Module name not found."
+                              " Check module Makefile syntax.")
     for r, d, f in os.walk(srcdir):
         for filename in f:
             if filename == "Makefile":
                 # get the module name: PGM = <module name>
                 with open(os.path.join(r, 'Makefile')) as fp:
                     for line in fp.readlines():
-                        if "PGM =" in line:
-                            modulename = line.split('=')[1].strip()
-                            module_list.append(modulename)
+                        if "PGM" in line:
+                            try:
+                                modulename = line.split('=')[1].strip()
+                                if modulename:
+                                    module_list.append(modulename)
+                                else:
+                                    grass.error(module_name_not_found)
+                            except IndexError:
+                                grass.error(module_name_not_found)
```

Comment:
This looks helpful, but unrelated to the GitLab from PR title and other changes. Can you put this separately. Especially for this change, note the tests in testsuite directory which you can run and also extend to test the behavior now and in the future.

Reply:
Yes, of course I can. I used the following gitlab extension url during testing `https://gitlab.com/vpetras/r.example.plus`. The installation was successful, but the module was not registered (checked with `g.extension -a`). Because missing space around '=' in the row in Makefile `PGM=r.example.plus` -> `PGM = r.example.plus`.